garak/_config.py

```diff
@@ -158,11 +158,12 @@ def load_config(
 
 def parse_plugin_spec(
     spec: str, category: str, probe_tag_filter: str = ""
-) -> List[str]:
+) -> tuple[List[str], List[str]]:
     from garak._plugins import enumerate_plugins
 
     if spec is None or spec.lower() in ("", "auto", "none"):
-        return []
+        return [], []
+    unknown_plugins = []
```

Comment:
Variable name `unknown_plugins` suggests the thing entered is a plugin, which isn't true if it's not found - can this be something like `unknown_clauses` instead?